src/tools/tidy/src/x.rs

```diff
@@ -0,0 +1,19 @@
+use std::process::Command;
+
+pub fn check(_bad: &mut bool) {
+    let result = Command::new("x")
+        .arg("--version")
+        .output();
+    let output = match result {
+        Ok(output) => output,
+        Err(_e) => todo!(),
```

Comment:
There two ways `output` can return an error:
- if `spawn` fails, most likely because `x` isn't installed. this is usually not a problem; we don't expect all contributors to use the shim.
- if `output` fails because of weird OS-specific problems. this is a bug and I think we should fail tidy because of it.

Can you use `Command::new().spawn() /* error handling */ .wait_with_output()` instead so we can handle these cases differently?